test/nntrainer_test_util.cpp

```diff
@@ -243,3 +244,30 @@ bool getMiniBatch_val(float *outVec, float *outLabel, int *status) {
   F.close();
   return true;
 }
+
+/**
+ * @brief return a tensor filled with contant value with dimension
+ */
+nntrainer::Tensor constant(float value, unsigned int batch, unsigned channel,
+                           unsigned height, unsigned width) {
+  nntrainer::Tensor t(batch, channel, height, width);
+  return t.apply([value](float) { return value; });
```

Comment:
how about using `t.setValue()` than `t.apply()`?

Reply:
NBD to change it though, could I ask for the reason why?

Reply:
`t.setValue(float)` is just more readable IMO :)